en/Getting started/Sync your notes across devices.md

```diff
@@ -113,3 +113,8 @@ Since Obsidian stores notes in a local folder on your Android device, you can al
 - [FolderSync](https://play.google.com/store/apps/details?id=dk.tacit.android.foldersync.lite)
 
 **Note:** Obsidian creates an Obsidian folder in the shared Documents folder. Any folder under `Documents/Obsidian` is considered an Obsidian vault.
+
+> [!important] Dealing with cloud-based files (e.g., OneDrive Files On-Demand / iCloud Drive optimized storage / Dropbox online-only files)
```

Comment:
Yeah, I don't know the best way to specify these files. It seems like every cloud service offers this feature a little differently. 

Maybe the headline should be "Cloud storage-based files will not work with Obsidian" or similar?

Reply:
> "Cloud storage-based files will not work with Obsidian"

Hm. Rather than cloud storage in general, since this callout is really about a common feature in cloud storage where are available as you use them, maybe something like "On-demand cloud storage"?

Reply:
Sure, works for me. I don't think there's a perfect phrase for this one, anyway.